persistence/db.go

```diff
@@ -153,24 +161,29 @@ func connectToDatabase() (*sql.DB, error) {
 	CREATE TABLE IF NOT EXISTS Tweets(Id SERIAL PRIMARY KEY, TweetTime text, Likes int, UserID int, Username text, Message text);
 	CREATE TABLE IF NOT EXISTS UserData(Id SERIAL PRIMARY KEY, Username text, Credentials text);
 	CREATE TABLE IF NOT EXISTS LikedTweets(Id int, UserID int);
+	CREATE TABLE IF NOT EXISTS BlockedUser(Id int, BlockedID int);
```

Comment:
Die Namen "UserId" und "TweetId" würde hier das Verständnis vereinfachen. In der Zeile drüber das gleiche.